packages/now-static-build/.gitignore

```diff
@@ -1 +1,5 @@
 dist/
+
+# bypass all ignored files for the cache fixtures
+# because they contain node_modules and package-lock.json files
+!test/cache-fixtures/**
```

Comment:
I'm seeing empty files. Is that right? 🤔 

![image](https://user-images.githubusercontent.com/229881/69370358-d1da4c80-0c6b-11ea-88e6-17e1125c5c81.png)

Reply:
Yes, these are fixtures 👍